src/provider.ts

```diff
@@ -80,7 +80,7 @@ export function detectProvider (userInput?: string) {
     return userInput
   }
 
-  if (process.env.VERCEL) {
+  if (process.env.VERCEL || process.env.NUXT_ENV_VERCEL_ENV) {
```

Comment:
Can you check if legacy `NOW_BUILDER` also exists? (in case nuxt template not detected)

Reply:
In fact, it's the only variable that seems to be set. 🙈